src/ChatWindow.js

```diff
@@ -168,6 +168,8 @@ class ChatWindow extends Component {
     if (this.props.authenticationInfo) {
       access_token = this.props.authenticationInfo.getAccessToken();
     }
+
+    // TODO: Do not make a call to avs if access_token is undefined.
```

Comment:
// TODO: Do not make a call to avs if access_token is undefined. We should somehow take the user back to login screen.